src/Runner.Worker/JobExtension.cs

```diff
@@ -321,7 +321,21 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
 
                     if (message.Variables.TryGetValue("system.workflowFileFullPath", out VariableValue workflowFileFullPath))
                     {
-                        context.Output($"Uses: {workflowFileFullPath.Value}");
+                        var usesOutput = $"Uses: {workflowFileFullPath.Value}";
+                        if (message.Variables.TryGetValue("system.workflowFileRef", out VariableValue workflowFileRef)
+                            && message.Variables.TryGetValue("system.workflowFileSha", out VariableValue workflowFileSha))
```

Comment:
```suggestion
                            || message.Variables.TryGetValue("system.workflowFileSha", out VariableValue workflowFileSha))
```

should it be `OR`?

Reply:
oh yea you are right :-) 
Just to note, we are sending both values but in case the user is using reusable workflow using sha then `workflowFileRef` will be empty. Still that will return `true` for this `TryGetValue`.